src/agent/onefuzz-agent/src/tasks/coverage/recorder.rs

```diff
@@ -136,6 +136,25 @@ impl CoverageRecorder {
 
         let mut cmd = Command::new("cdb.exe");
 
+        // Unless users explicitly set symbol path, don't perform external symbols
+        // lookup during sancov extraction
+        let mut target_env = self.config.target_env.clone();
+        let cache_path = format!(
+            "cache*{}",
+            self.config
+                .target_exe
+                .parent()
+                .ok_or_else(|| format_err!("invalid path parent"))?
+                .join("symbols")
```

Comment:
What happens if the setup container has a pre-existing `symbols` dir that isn't well-formed as a WinDBG cache directory?

More generally, I thought we just wanted to set the `setup` dir as one of the _search_ paths, not the root of a _cache_ directory. Do we actually want the `cache*` part of this?